### PR TITLE
ConstraintViolationException is missing annotation information with default validator.

### PR DESCRIPTION
**Pull request by traycho**
Once receiving ConstraintViolationException raised by  default validator the annotation information is lost.

### CODE REVIEW

File: router/src/main/java/io/micronaut/web/router/Router.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
      */
     default @Nonnull <T, R> Stream<UriRouteMatch<T, R>> findAny(@Nonnull CharSequence uri, @Nullable HttpRequest<?> context) {
         return findAny(uri);
-    };
+    }
```
Comment (traycho) [inline router/src/main/java/io/micronaut/web/router/Router.java]: Fix checkstyle  from previous merged PR in router project.

File: validation/src/main/java/io/micronaut/validation/validator/DefaultConstraintDescriptor.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
+/*
```
Comment (traycho) [inline validation/src/main/java/io/micronaut/validation/validator/DefaultConstraintDescriptor.java]: This class was internal for IntrospectedBeanDescriptor,  just exposed, no real code change here.

File: validation/src/main/java/io/micronaut/validation/validator/DefaultConstraintDescriptor.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+import java.util.stream.Collectors;
+
+/**
+ * Default constraint descriptor implementation.
+ *
+ * @param <T> The constraint type
+ * @author graemerocher
+ * @since 1.2
+ */
+class DefaultConstraintDescriptor<T extends Annotation> implements ConstraintDescriptor<T> {
```
Comment (jameskleeh) [inline validation/src/main/java/io/micronaut/validation/validator/DefaultConstraintDescriptor.java]: This should be annotated with `@Internal`

**Comment by traycho** [comment]
Build is failing because of test test returning a file from a controller, I think it is broken in 1.2.x.

**Comment by graemerocher** [comment]
Thanks for the contribution

**Pull request merged**